moodle/Tests/FilesBoilerPlateCommentTest.php

```diff
@@ -92,6 +92,19 @@ public function test_moodle_files_boilerplatecomment_short() {
         $this->verify_cs_results();
     }
 
+    public function test_moodle_files_boilerplatecomment_short_empty() {
+            $this->set_standard('moodle');
```

Comment:
Thanks, @stronk7. This looks spot-on to me.  The indentation of this part seems to need some fixing, though.

Reply:
Whops, it should be fixed now. Thanks!